app/src/main/kotlin/com/intive/picover/parties/view/PartiesScreen.kt

```diff
@@ -56,8 +56,8 @@ fun PartiesScreen(
 		MVIState.Type.LOADING -> PicoverLoader(modifier = Modifier.fillMaxSize())
 		MVIState.Type.LOADED -> LoadedContent(
 			parties = state.parties,
-			onPartyClick = viewModel::onPartyClick,
-			onFabClick = viewModel::onAddPartyClick,
+			onPartyClick = { viewModel.emitEffect(NavigateToPartyDetails(it)) },
+			onFabClick = { viewModel.emitEffect(NavigateToAddParty) },
 		)
 
 		MVIState.Type.ERROR -> PicoverGenericError(onRetryClick = { viewModel.emitEvent(PartiesEvent.Load) })
```

Comment:
Comment to restricted visibility of `MVIViewModel::handleEvent`
Please always remember about visibility of functions and properties in basic classes, we shouldn't expose too much. It prevents from unintended usage of our class and makes more clear how it should be used. 
In this case, we could call `viewModel.handleEvent(PartiesEvent.Load)`, it would be even faster and more efficient :)